specification/cognitiveservices/data-plane/Language/preview/2022-10-01-preview/analyzetext.json

```diff
@@ -752,7 +752,7 @@
     "ClassificationDocumentResult": {
       "type": "object",
       "properties": {
-        "class": {
+        "classifications": {
```

Comment:
I don't think you can change property name already in public preview.

Reply:
this is a bug in the swagger, the actual served content has always been classifications, so no breaking change there (SDKs have not released yet and they need this change to be able to release), plus this feature is gated private preview and we haven't allowed any customers yet to use it.

Reply:
Ok. I think I get the context. I will let Heath or others to sign off.

Meanwhile, check errors in ModelValidation